README.md

```diff
@@ -3,4 +3,9 @@
 Check the Github Actions workflow for how to build the code.
 Check https://github.com/solvcon/mmnote for the theoretical and numerical notes.
 
+## Binaries
+A portable version of the modmesh viewer for Windows is now available for download from the Actions: [**devbuild**](https://github.com/solvcon/modmesh/actions/workflows/devbuild.yml?query=event%3Aschedule+is%3Asuccess+branch%3Amaster) (login required).
+
+> Click on a run result and scroll down to the Artifacts section to download the ZIP file.
```

Comment:
If add a clearly indicating to click `build_windows-2022_Release` and `archive_portable_artifacts` should help users find the download url more quickly.

Reply:
I'm not sure if a two-step click from the job `build_windows-2022_Release` to the step `archive_portable_artifacts` is more intuitive than scrolling down to the Artifacts section with just one click in the workflow.